src/pamela/parser.clj

```diff
@@ -543,9 +558,7 @@
                 :pclass-ctor ir-pclass-ctor
                 :pclass-ctor-arg identity
                 :pclass-ctor-option identity
-                :pclass-name identity
```

Comment:
Other than test/cli/30_circuit.txt, is/was there any use of `:pclass-name` or `:plant-fn-symbol`?  I didn't notice any IR changes.